src/Poke-API/api.jsx

```
@@ -5,14 +5,17 @@ import { Form, Button, Card, Modal, ProgressBar } from 'react-bootstrap';
 const PokeApi = () => {
   const [query, setQuery] = useState('');
   const [pokemon, setPokemon] = useState(null);
+  const [page, setPage] = useState(1);
   const [recommendedPokemon, setRecommendedPokemon] = useState([]);
   const [error, setError] = useState('');
```

ESLint

[error] 'error' is assigned a value but never used.

(no-unused-vars)